index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="index.css" rel="stylesheet" />
+    <link
+      href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    
+    <title>Capstone1</title>
+  </head>
+  <body>
+    <header>
+      <nav class="tools">
+        <img src="images/facebook-icon.png" alt="" class="fb_icon">
+        <img src="images/twitter-icon.png" alt="" class="fb_icon">
+        <a href="#" class="english">English</a>
+        <a href="#" class="english">My page</a>
+        <a href="#" class="english" id="last_tool">Logout</a>
+      </nav>
+      <nav id="NavMobile">
+        <img src="images/Hamburger.png" alt="" class="hamburger" id="hamburger1"/>
+        <img src="images/x-icon.png" alt="" class="hamburger" id="hamburger2"/>
+        <ul class="ul_nav" id="ul_nav">
+          <li id="home"><a class="links_Nav" >Home</a></li>
+          <li id="about"><a class="links_Nav" >About</a></li>
+          <li><a class="links_Nav" >Join</a></li>
+          <li><a class="links_Nav" href="#">Program</a></li>
+          <li><a class="links_Nav" href="#">Sponsor</a></li>
+          <li><a class="links_Nav" href="#">News</a></li>
+        </ul>
+        <!-- <img src="images/PngItem_1074369.png" alt="" class="world_img" /> -->
+      </nav>
+      <nav id="NavDesck">
+        <div class="logo_div_Nav" id="lodNav">
+          <img src="images/v-icon-3.jpg" alt="" class="img_logo2">
+          <h2 class="title_footer">The Vitron community</h2>
+        </div>
+        <ul id="Ul_desk">
+          <li id="home2"><a class="links_Nav" >Home</a></li>
+          <li id="about2"><a class="links_Nav" >About</a></li>
+          <li><a class="links_Nav" >Join</a></li>
+          <li><a class="links_Nav" href="#">Sponsor</a></li>
+          <li><a class="links_Nav" href="#">News</a></li>
+          <li><a class="links_Nav" id="Ignite" href="#">V Ignite</a></li>
+        </ul>
+        <!-- <img src="images/PngItem_1074369.png" alt="" class="world_img" /> -->
+      </nav>
+     
+    </header>
+
+    <section class="home_page" id="home_page">
+      <div class="section_intro">
+        <h2 class="Intro">"Hello, make it bounce with Vitron"</h2>
+        <h1 class="big_title">THE VITRON COMMUNITY SAYS HELLO.</h1>
```

Comment:
- [ ] Kindly decrease the font size of this heading in the mobile version `about` page and make it in the center to match the Behance design and to get rid of the general scrolling problem.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/163586744-ad3dfe54-0071-4af8-af43-ba0504ea30b7.png) | ![image](https://user-images.githubusercontent.com/24703171/163586378-59e8241c-79a0-4916-8582-d76f7060ab8a.png) |
 
![image](https://user-images.githubusercontent.com/24703171/163589369-caca1614-5a13-4e35-891d-199058a63bd2.png)